### PR TITLE
Cheryl-Mauro blog skeleton

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Comment by yabellini** [comment]
Hi @cherylisabella and @maurolepore, this is the draft of the blog pot for introducing you both.  Please, review and made the changes you consider.  You can add some links to the organization, initiatives you mention in your text.  You can also change it or add more details.

File: content/blog/2023-05-30-champions-program-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+</br>
+
+## Mauro Lepore - Mentor
+
+{{< figure src = "/img/team/mauro-lepore-mentor.jpg" class = "pull-left" width = "300" alt = "Profile photo of Mauro Lepore" caption = "<center><strong>Mauro Lepore</strong></center>">}}
+
+I’m Mauro Lepore. I grew up in Argentina, studied biology at the University of Buenos Aires, then did a PhD in coral reefs ecology at the University of Queensland, Australia.
+
+I’m now a research software developer at 2 Degrees Investing Initiative, and an editor for rOpenSci. I’m excited to join the rOpenSci Champions Program as a mentor.
```
Comment (maurolepore) [inline content/blog/2023-05-30-champions-program-teams/index.md:41]:
```suggestion
I’m now a research software developer at [2 Degrees Investing Initiative](https://2degrees-investing.org/), and an editor for rOpenSci. I’m excited to join the rOpenSci Champions Program as a mentor. 
```

**Review by maurolepore — APPROVED** [review]
LGTM. Thanks @yabellini!

**Comment by yabellini** [comment]
@steffilazerte another one ready for your review. Thanks!

**Review by steffilazerte — COMMENTED** [review]
Looks good, just a couple tweaks and missing an author file for Cheryl Isabella.

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok
* [X] alternative text of images is informative
* [ ] author metadata is provided with correct folder name -> Cheryl Isabella needs an author file
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

**Pull request merged**